src/Jackett.Common/Definitions/lacale-api.yml

```diff
@@ -0,0 +1,159 @@
+    # Software / Logiciels
+    - {id: software, cat: PC, desc: "Logiciels"}
+    - {id: windows, cat: PC, desc: "Windows"}
+    - {id: linux, cat: PC, desc: "Linux"}
+    - {id: mac, cat: PC/Mac, desc: "Mac"}
+    - {id: pc, cat: PC, desc: "PC"}
+    # Games / Jeux
+    - {id: games, cat: PC/Games, desc: "Jeux"}
+    - {id: consoles, cat: Console, desc: "Consoles"}
+    # XXX
+    - {id: xxx, cat: XXX, desc: "XXX"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep]
+    movie-search: [q]
+    music-search: [q]
+    book-search: [q]
+
+settings:
+  - name: passkey
+    type: text
+    label: Passkey
+  - name: info_key
+    type: info
+    label: About your Passkey
+    default: "Find your Passkey on <a href=\"https://la-cale.space/faq/api\" target=\"_blank\">La Cale > API page</a>."
+  - name: freeleech
+    type: checkbox
+    label: Search freeleech only
+    default: false
+  - name: multilang
+    type: checkbox
+    label: Replace MULTi by another language in release name
+    default: false
+  - name: multilanguage
+    type: select
+    label: Replace MULTi by this language
+    default: FRENCH
+    options:
+      FRENCH: FRENCH
+      MULTi.FRENCH: MULTi.FRENCH
+      ENGLISH: ENGLISH
+      MULTi.ENGLISH: MULTi.ENGLISH
+      VOSTFR: VOSTFR
+      MULTi.VOSTFR: MULTi.VOSTFR
+  - name: vostfr
+    type: checkbox
+    label: Replace VOSTFR and SUBFRENCH with ENGLISH
+    default: false
+
+login:
+  path: api/external
+  method: get
+  inputs:
+    passkey: "{{ .Config.passkey }}"
+  error:
+    - selector: ":root:contains(\"Invalid passkey\")"
+      message:
+        text: "The Passkey was not accepted by {{ .Config.sitelink }}."
+    - selector: ":root:contains(\"Unauthorized\")"
+      message:
+        text: "The Passkey was not accepted by {{ .Config.sitelink }}."
+
+search:
+  paths:
+    - path: api/external
+      inputs:
+        passkey: "{{ .Config.passkey }}"
+        cat: "{{ if .Categories }}{{ join .Categories \",\" }}{{ else }}{{ end }}"
+        q: "{{ .Keywords }}"
+      response:
+        type: json
+  rows:
+    selector: $
+
+  fields:
+    categorydesc:
+      selector: category
+    title_phase1:
+      selector: title
+    _title:
+      selector: title
+    title_vostfr:
+      text: "{{ .Result.title_phase1 }}"
+      filters:
+        - name: re_replace
+          args: ["(?i)\\b(vostfr|subfrench)\\b", "ENGLISH"]
+    title_phase2:
+      text: "{{ if .Config.vostfr }}{{ .Result.title_vostfr }}{{ else }}{{ .Result.title_phase1 }}{{ end }}"
+    title_multilang:
+      text: "{{ .Result.title_phase2 }}"
+      filters:
+        - name: re_replace
+          args: ["(?i)\\b(MULTI(?!.*(?:FRENCH|ENGLISH|VOSTFR)))\\b", "{{ .Config.multilanguage }}"]
+    title:
+      text: "{{ if .Config.multilang }}{{ .Result.title_multilang }}{{ else }}{{ .Result.title_phase2 }}{{ end }}"
+    details:
+      selector: guid
+      filters:
+        - name: prepend
+          args: "{{ .Config.sitelink }}torrents/"
```

Comment:
This isn't working for me, details links are the title made URL-friendly, followed by some random characters